src/components/EventList/components/EventItem/index.jsx

```diff
@@ -47,6 +53,7 @@ const EventItem = ({ event }) => {
               timeZone="currentBrowser"
               description={event.Description}
               onClick={() => setExpanded((e) => !e)}
+              lightMode="system" // Set to system, if we give users
```

Comment:
So, this still needs some work, and I'm not sure how to solve it to be honest...  The add-to-calendar button just takes a mode.  In system, it will be just set to the users system setting, but it has no way of knowing what the 360 local set mode is.  So, for users who are set to system in 360, and those who have their system set the same as 360, the button will appear correct, but otherwise the button might be light in dark mode and vice versa.

I'm thinking of trying to create a global function or hook or something to check if 360 is in dark mode within javascript files, but it doesn't seem like there is an easy way to do that in MUI.  

For now, it works in some cases, and doesn't look to terrible otherwise, and I'm aware of the issue, but I can add a TODO comment to the code here if that is helpful @russtuck

Reply:
Your explanation here is really clear.  Please put it in a comment, so it will stay with the code.  (You can include a TODO comment about possibly doing something better.)